README.md

```diff
@@ -70,6 +70,9 @@ Display BPMN Diagram:
 - [BPMN Diagram fit on load](examples/diagram-navigation/diagram-fit-on-load/README.md) - fit the BPMN diagram in the container on load
 - [BPMN Diagram fit after load](examples/diagram-navigation/diagram-fit-after-load/README.md) - fit the BPMN diagram in the container after load
 
+### Overlay examples
+- [Add and remove overlays](examples/overlays/add-remove/README.md) - add and remove overlays on BPMN elements
```

Comment:
❓ can we mention this example is all about usage of the default overlay (we currently have only one) and positioning the overlay?
This will prevent to have to update this example when new will be added about styling, overlay type selection, ...